artemis-server/src/main/java/org/apache/activemq/artemis/core/paging/impl/PagingManagerImpl.java

```diff
@@ -437,12 +449,27 @@ public void processReload() throws Exception {
       }
    }
 
+   private boolean ignoreGlobalMaxSize(SimpleString address) {
+      if (this.addressPrefixesIgnoringGlobalMaxSize == null) {
+         return false;
+      } else {
+         //for a small number of prefixes to check we can use just a linear search too :)
+         for (SimpleString prefixes : this.addressPrefixesIgnoringGlobalMaxSize) {
```

Comment:
ATM it will have an array of just 1 element

Reply:
why even code this to be more than one, concern here would be what else? could end up in non-managed space? Its a dangerous thing to have anything unbounded entirely. See my comment about possible alternative and safer approach with implementing a two tier global max sizing, one global  (all addresses) and one non-management max which would be set lower to allow head room for the management ones no matter what (all addresses except management)

Reply:
I have implemented this feature lefting it opened to be extended to allow other address prefixes to be added too, but I can implementing it too just considering this specific use case too: just magement addresses messages.

Reply:
I taken the point and allowed that address Set to be just the one I need (if any), lefting the code be simple as it needs to be.